pyproject.toml

```diff
@@ -47,7 +47,11 @@ name = "Frequenz Energy-as-a-Service GmbH"
 email = "floss@frequenz.com"
 
```

Comment:
Thanks, will fix!

Reply:
Which commit exactly? I can't find any with an extra `i`. If it is `Add missing dependency to tomli` then the dependency is actually called `tomli` with the trailing `i`, so that one is correct.